.github/workflows/tf-apply-hash.yml

```diff
@@ -0,0 +1,129 @@
+name: HASH Terraform Apply
+
+# TODO: While this workflow is specific to HASH, we should consider
+#  making this a reusable workflow, parameterized on the directory and environment.
+
+on:
+  workflow_dispatch:
+    inputs:
+      env:
+        type: choice
+        options: ["prod"]
```

Comment:
What is the reason we don't allow dispatching `dev`?

Reply:
We don't have a prod HASH environment :)

Reply:
I assume you mean `dev` but then the comment below in `matrix-eval` needs to be adjusted 🙂

Reply:
Yes sorry a dev env. I will adjust the comments!